### PR TITLE
Created Information Section-3

### LINKED ISSUE #10: Create Information Section-3
_As a site visitor, I can see new information and I am entertained by the comical images_

> Assigned to `Yana`

### Branch(es)

- [x] Create an issue related to this user story on Github and assign self. 
- [x] Add a brach named `info-3-no-tracking`
- [x] After html and css are done, commit
- [x] Adjust `development-strategy.md` for next development, if needed
- [x] Push the changes
- [x] Pull request and merge it on Github

### HTML

- [x] Use section to explain how the service of DuckDuckGo work: no tracking on your searches
- [x] Add comical image to entertain visitors
- [x] Add button to make sure they are able to access this service directly

### CSS

- [x] Add styling on new elements

### PR DESCRIPTION
**Pull request by adekimpianna**
It contains text and image for information section: no tracking for the search.

closes #10

### CODE REVIEW

**Comment by adekimpianna** [comment]
situation before solving conflict: branch only has their own code   
afbeelding

**Review by Boam1994 — APPROVED** [review]

**Comment by adekimpianna** [comment]
The branch is updated with master https://htmlpreview.github.io/?https://github.com/harunaltunhr/duck-duck-clone/blob/info-3-no-tracking/index.html

Master (yellow-green) with section info 3 (purple)
afbeelding

File: style.css [hunk 1]
```diff
@@ -2,7 +2,6 @@ body{
     font-family: "DDG_ProximaNova","DDG_ProximaNova_UI_0","DDG_ProximaNova_UI_1","DDG_ProximaNova_UI_2","DDG_ProximaNova_UI_3","DDG_ProximaNova_UI_4","DDG_ProximaNova_UI_5","DDG_ProximaNova_UI_6","Proxima Nova","Helvetica Neue","Helvetica","Segoe UI","Nimbus Sans L","Liberation Sans","Open Sans",FreeSans,Arial,sans-serif;
     color: #222;
     line-height: 1.6;
-
```
Comment (harunaltunhr) [inline style.css:5]: ??? Why did you change body style?

**Review by harunaltunhr — COMMENTED** [review]
@adekimpianna you changed body style, will we change it from all branch?

**Comment by adekimpianna** [comment]
@harunaltunhr I deleted the overflow (:hidden you made on initial master) when solving conflict for the second PR. Now Second PR is merged, so I believe the new remote master doesn’t have that overflow anymore. When you pull updates from remote, and the overflow: hidden is still there, then you can arrange all accordingly when solving conflict to make the page works, as Evan and Stephane explained on Zoom.

File: style.css (hunk 1 above)
Comment (adekimpianna) [reply style.css:5]: Because you made `overflow:hidden` on the initial commit, all the other elements that overflow that screen dimension are hidden. If we dont delete that `overflow:hidden`, we can only see the screen that show the top html elements. In this project we need to see the whole pages, we dont need to hide them. So I had to delete that so that we can see all our other elements on that looooong page, so we can scroll down the page.
